Change fonts (#149)
opengisch/QField-docs: en/prepare/change-fonts.rst

> Using specific fonts

Reviewer: ```suggestion
Using additional fonts

```

=====================

In QField, you are able to use all the fonts you want.

> There are two different possibilities to do it:

Reviewer: ```suggestion
There are two different possibilities to register additional fontst:

```

> 1. By adding the fonts (.ttf or .otf) in the device in the folder `<device root>/QField/fonts`.

Reviewer: ```suggestion
1. By adding fonts (.ttf or .otf) on the device in the folder `<device root>/QField/fonts`; those will be made accessible to all projects and individual datasets.

```

> 2. By adding the fonts (.ttf or .otf) in the same folder as the QGIS project (.qgs or .qgz).

Reviewer: ```suggestion
2. By adding fonts (.ttf or .otf) in the same folder as a given project file (.qgs or .qgz); those will be accessible only when viewing that project file.

```
